ckanext/canada/templates/public/base.html

```diff
@@ -37,7 +37,7 @@
 {%- endblock -%}
 
 {% block feedback_survey %}
-  <script async defer type="text/javascript" src="{{ h.url_for_static('/static/js/foresee_survey.js') }}" ></script>
+  <script src="/themes/custom/gcweb/js/foresee-survey.min.js" defer="" async=""></script>
```

Comment:
how about adding a comment here like `{# include forsee survey js from drupal hosted on the same domain #}` because we generally don't hard-code url paths into our templates

Reply:
how about I define a variable in settings ini file and use it here?

Reply:
that sounds even better "canada.survey_js_url" or similar would act as documentation and would make it easy to enable/disable from the config file